addons/tmf_loadouts/loadouts/us_army_2003.hpp

```diff
@@ -0,0 +1,544 @@
+		"cup_optic_compm2_low",
+		LIST_2("")
+	};
+	attachment[] = {
+		"cup_acc_anpeq_2_grey",
+		"cup_acc_anpeq_2_black_top"
+	};
+	magazines[] = {
+		LIST_7("CUP_30Rnd_556x45_Stanag"),
+		LIST_2("CUP_30Rnd_556x45_Stanag_Tracer_Red"),
+		LIST_2("HandGrenade"),
+		"SmokeShell"
+	};
+};
+
+class g : r {
+	displayName = "Grenadier";
+	vest[] = {
+		LIST_3("CUP_V_B_Interceptor_Grenadier_M81"),
+		"usm_vest_pasgt_lbe_gr"
+	};
+	primaryWeapon[] = {
+		"CUP_arifle_M4A1_BUIS_GL",
+		"CUP_arifle_M4A1_GL_carryhandle"
+	};
+	attachment[] = {
+		"cup_acc_anpeq_2_grey",
+		"cup_acc_anpeq_2_black_top"
+	};
+	magazines[] += {
+		LIST_11("CUP_1Rnd_HEDP_M203"),
+		LIST_2("CUP_1Rnd_Smoke_M203")
+	};
+};
+
+class car : r {
+	displayName = "Carabinier";
+	magazines[] = {
+		LIST_7("CUP_30Rnd_556x45_Stanag"),
+		"HandGrenade",
+		"SmokeShell"
+	};
+};
+
+class cls : car {
+	displayName = "Combat Life Saver";
+	traits[] = {"medic"};
+	vest[] = {
+		"CFP_US_pasgt_alice_Medic_ERDL",
+		"CUP_V_B_Interceptor_Rifleman_M81"
+	};
+	backpack[] = {"CUP_B_AlicePack_OD"};
+	backpackItems[] = { MEDICAL_CLS };
+};
+
+class m : cls {
+	displayName = "Medic";
+	backpack[] = {"usm_pack_alice"};
+	backpackItems[] = { MEDICAL_M };
+};
+
+class smg : r {
+	displayName = "Submachinegunner";
+	vest[] = {"CUP_V_B_Interceptor_Base_M81"};
+	primaryWeapon[] = {"CUP_smg_MP5A5"};
+	magazines[] = {
+		LIST_8("CUP_30Rnd_9x19_MP5"),
+		"HandGrenade",
+		"SmokeShell"
+	};
+};
+
+class ftl : g {
+	displayName = "Fireteam Leader";
+	backPack[] = {"CUP_B_SLA_Medicbag"};
+	backpackItems[] += {"ACE_MapTools"};
+	magazines[] += {
+		LIST_2("1Rnd_SmokeRed_Grenade_shell")
+	};
+	linkedItems[] += {
+		"Binocular",
+		"ItemGPS"
+	};
+};
+
+class sl : ftl {
+	displayName = "Squad Leader";
+	vest[] = {
+		"CUP_V_B_Interceptor_Rifleman_M81",
+		"CFP_US_pasgt_alice_SquadLeader_ERDL"
+	};
+	backPack[] = {"CUP_B_SLA_Medicbag"};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	magazines[] += {
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+	linkedItems[] += {
+		"ACE_Yardage450",
+		"ItemGPS"
+	};
+};
+
+class co : sl {
+	displayName = "Platoon Leader";
+	headgear[] = {"usm_bdu_8point_dcu"};
+	vest[] = {
+		"CFP_US_pasgt_alice_officer_ERDL",
+		"CUP_V_B_Interceptor_Base_M81"
+	};
+	backPack[] = {"CUP_B_SLA_Medicbag"};
+};
+
+class fac : co {
+	displayName = "Forward Air Controller";
+	backpack[] = {"usm_pack_alice_prc119"};
+	backpackItems[] = { FAC_GEAR };
+	linkedItems[] += {
+		"CUP_SOFLAM",
+		"ItemGPS"
+	};
+};
+
+class ar : r {
+	displayName = "Automatic Rifleman";
+	vest[] = {
+		"CFP_US_pasgt_alice_SAW_ERDL",
+		"CUP_V_B_Interceptor_Base_M81"
+	};
+	backPack[] = {"CUP_B_AlicePack_Khaki"};
+	primaryWeapon[] = {"CUP_lmg_M249_E2"};
+	sidearmWeapon[] = {"CUP_hgun_M9"};
+	magazines[] = {
+		LIST_4("CUP_200Rnd_TE4_Red_Tracer_556x45_M249"),
+		"HandGrenade",
+		"SmokeShell",
+		LIST_3("CUP_15Rnd_9x19_M9")
+	};
+};
+
+class aar : r {
+	displayName = "Assistant Automatic Rifleman";
+	backPack[] = {"usm_pack_200rnd_556_bandoliers"};
```

Comment:
No AR ammo

Reply:
USM (apparently) has a script that runs that fills the USM Bandoliers/Medic Backpacks with the appropriate supplies. Found that out when testing in the editor lol.

Reply:
#38